src/angular/styles/_typography.scss

```diff
@@ -750,6 +732,23 @@
       --sbb-color-weak-accent: var(--sbb-color-milk);
       --sbb-color-error: var(--sbb-color-red_dark-mode);
 
+      // Checkbox & Radiobutton
+      --sbb-selection-container-background-color-default: var(--sbb-color-background);
+      --sbb-selection-container-background-color-disabled: var(--sbb-color-charcoal);
+      --sbb-selection-container-border-color-disabled: var(--sbb-color-smoke);
+      --sbb-selection-container-border-color-focused: var(--sbb-color-text);
+      --sbb-selection-container-icon-color-default: var(--sbb-color-text);
+      --sbb-selection-container-icon-color-disabled: var(--sbb-color-smoke);
+      --sbb-selection-label-color-checked: var(--sbb-color-text);
+      --sbb-selection-label-color-unchecked: var(--sbb-color-graphite);
+      --sbb-selection-label-color-disabled: var(--sbb-color-smoke);
+
+      // Context colors
+      --sbb-color-background: var(--sbb-color-black);
```

Comment:
Context colors for dark mode are already defined above. Please remove them here.